test/zipWith.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+var S = require('..');
+
+var eq = require('./internal/eq');
+
+
+test('zipWith', function() {
+
+  eq(typeof S.zipWith, 'function');
+  eq(S.zipWith.length, 3);
+  eq(S.zipWith.toString(), 'zipWith :: (a -> b -> c) -> Array a -> Array b -> Array c');
+
+  eq(S.zipWith(S.add, [1, 2, 3, 4], [3, 2, 1]), [4, 4, 4]);
+  eq(S.zipWith(S.add, [1, 2, 3], [3, 2, 1, 0]), [4, 4, 4]);
+  eq(S.zipWith(S.gte, [1, 2, 3], [3, 2, 1]), [true, true, false]);
+  eq(S.zipWith(a => b => [a, b], [1, 3, 5], [2, 4, 6]), [[1, 2], [3, 4], [5, 6]]);
```

Comment:
You may have noticed that the source code and test suite do not use any ES6 syntax. This is intentional. You'll need to use `function(a) { ... }` here. ES6 syntax can, though, be used in doctests because we do not run doctests in browsers.

Reply:
Thanks! I'll do that